css/css-conditional/at-supports-48.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+  <head>
+    <title>CSS Test (Conditional Rules): In @supports, ::file-selector-button can be parsed successfully</title>
+    <link rel="author" title="Tim Nguyen" href="https://github.com/nt1m">
+    <link rel="help" href="http://www.w3.org/TR/css3-conditional/#at-supports">
+    <link rel="match" href="at-supports-001-ref.html">
+    <style>
+      div {
+        background: red;
+        height: 100px;
+        width: 100px;
+      }
+      @supports selector(input::file-selector-button) {
```

Comment:
It would be nice to add a link to whatever spec defines `::file-selector-button`, because I don't know what spec defines that off-hand.

Reply:
Done